app/models/foreman_tasks/task.rb

```diff
@@ -255,6 +255,33 @@ def action_continuous_output
       main_action.continuous_output.raw_outputs
     end
 
+    def self.get_latest_tasks_by_resource_ids(label, resource_ids, resource_type)
```

Comment:
Could we drop the `get_` prefix?

Could we swap the `resource_type` and `resource_ids` args so it reads more like "give me all tasks with $label which are related to $resource_type with ids $resource_ids"?